script.js

```diff
@@ -76,7 +76,6 @@ const sectionsCard = [
     btnImg: 'images/Vector github.png',
     btnImg2: 'images/Icon.png',
   },
-
   {
     title: 'Multi-Post Stories',
     list: ['Canopy', 'Back end Dev', '2015'],
```

Comment:
- [ ] I am highlighting these sets of codes because I can't highlight the codes that need to be removed. Kindly remove all `Eslint` disabled comments from this file (from line 1 to line 6). `Eslint` helps us to write clean, maintainable and readable codes and disabling it would defeat this purpose. Kindly remove them. 👍 

![image](https://user-images.githubusercontent.com/77106962/211164209-ed64adc2-2c23-4660-a4b2-94dbefe70149.png)

Reply:
Fixed.